### ISSUE TITLE
Over-complete progress bar

### ISSUE THREAD

**Issue by tmtmtmtm** [everypolitician/gender-balance#243]
![screen shot 2015-08-07 at 18 48 11](https://cloud.githubusercontent.com/assets/57483/9142069/42971652-3d35-11e5-9b58-f664d3478c53.png)

**Comment by andylolz** [comment]
I think happens as a result of pressing the buttons very quickly, thereby submitting multiple responses for the same politician. So it’s possible there is another bug here related to handling (or preventing) multiple responses.

**Comment by chrismytton** [comment]
Hmm, I'm still seeing this problem after deploying #247. Interestingly I also see it on Australia, so I wonder if there's something funny going on with calculations for countries with more than one legislature?

**Comment by chrismytton** [comment]
It looks like because the gender data for the Australian Representatives was added, so we're double counting some people, once for where people have "voted" on them, and again if we have gender data for them.

So if you completed all of the Australian Representatives and Senate (241 people) before we had any gender data on them, then we added gender data for all the Representatives (150) then [`percent_complete` would be double counting people](https://github.com/everypolitician/gender-balance/blob/fdde893d0e0a1843f3f4ab252afc2d20af0be67a/lib/helpers.rb#L33).

The fix is to exclude the ids of people that we already have gender for when we're calculating the `complete_people` variable in `percent_complete`.
